python/dgl/data/rdf.py

```diff
@@ -0,0 +1,566 @@
+"""RDF datasets
+
+Dataset from "A Collection of Benchmark Datasets for
+Systematic Evaluations of Machine Learning on
+the Semantic Web"
+"""
+import os, sys
+from collections import namedtuple, OrderedDict
+import itertools
+import rdflib as rdf
+import pickle
+import abc
+import re
+
+import networkx as nx
+import numpy as np
+
+import dgl
+import dgl.backend as F
+from .utils import download, extract_archive, get_download_dir, _get_dgl_url
+
+__all__ = ['AIFB', 'MUTAG', 'BGS', 'AM']
+
+class Entity:
+    def __init__(self, id, cls):
+        self.id = id
+        self.cls = cls
+
+    def __str__(self):
+        return '{}/{}'.format(self.cls, self.id)
+
+class Relation:
+    def __init__(self, cls):
+        self.cls = cls
+
+    def __str__(self):
+        return str(self.cls)
+
+class RDFGraphDataset:
+    """
+
+    Attributes
+    ----------
+    graph
+    num_classes
+    predict_category
+    train_idx
+    test_idx
+    labels
+    """
```

Comment:
Add description about the attributes, including type, shape, what they are and give examples if necessary.